test/integration/src/test/java/io/pravega/test/integration/ControllerStreamMetadataTest.java

```diff
@@ -100,7 +100,7 @@ public void tearDown() {
         }
     }
 
-    @Test(timeout = 60000)
+    @Test(timeout = 10000)
```

Comment:
60s doesn't seem to high and could be left as is.
How long does this test typically take? 
Since this is an integration test, hope 10s doesn't cause timeout in slow systems.